main.rb

```diff
@@ -0,0 +1,44 @@
+require_relative 'app'
+
+def display
+  puts 'Please choose an option by entering a number:'
+  puts '1 - List all books'
+  puts '2 - List all people'
+  puts '3 - Create a person'
+  puts '4 - Create a book'
+  puts '5 - Create a rental'
+  puts '6 - List all rentals for a given person id'
+  puts '7 - Exit'
+end
+
+def run_app(app)
+  menu_options = {
+    '1' => :list_all_books,
+    '2' => :list_all_peoples,
+    '3' => :create_person,
+    '4' => :create_book,
+    '5' => :create_rental,
+    '6' => :list_rentals,
+    '7' => :exit
+  }
+  loop do
+    display
+    input = gets.chomp
+
+    if menu_options.key?(input)
+      run = menu_options[input]
+      app.send(run)
+      break if run == 'exit'
+    else
+      puts 'Enter the correct option: '
+    end
+  end
+end
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:**  Good job by making your console application behaviors in the same way like the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ). However, I think it would be better if you add a message similar to this: `Thank You for using this app!!!` when a user exit the application. This will make your app matches the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ) at 100%.